packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditor.tsx

```diff
@@ -32,54 +28,48 @@ function _TableColumnSchemaEditor(props: TableColumnSchemaEditorProps) {
   const { entityId } = props
 
   const formRef = useRef<SubmitHandle>(null)
-  const { data: entity, isLoading: isLoadingEntity } = useGetEntity(entityId)
-  const { data: _queryResultBundle, isLoading: isLoadingColumnModels } =
-    useGetQueryResultBundleWithAsyncStatus(
-      {
-        entityId,
-        query: {
-          sql: `SELECT * FROM ${entityId}`,
-        },
-        partMask: BUNDLE_MASK_QUERY_COLUMN_MODELS,
-        concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
-      },
-      {
-        // This data is inserted into a form, so don't refetch it.
-        staleTime: Infinity,
-        useErrorBoundary: true,
-      },
-    )
+  const { data: bundle, isLoading } = useGetEntityBundle(
```

Comment:
The SWC implementation gets the column models from the bundle, rather than issuing a table query. Match that behavior--note that this code is currently unused outside of storybook.